pkg/internal/checkup/checkup.go

```diff
@@ -156,18 +156,10 @@ func (c *Checkup) Teardown(ctx context.Context) error {
 		return fmt.Errorf("%s: %w", errPrefix, err)
 	}
 
-	if err := c.deletePod(ctx); err != nil {
```

Comment:
what about the actual implementation? Don't you want to remove it as well?

Reply:
It is removed in the second commit.

Reply:
but it should be removed in the first one, no?